pkg/stanza/operator/parser/container/parser.go

```diff
@@ -117,35 +118,35 @@ func (p *Parser) Process(ctx context.Context, entry *entry.Entry) (err error) {
 			// parse the message
 			err = p.ParserOperator.ParseWith(ctx, entry, p.parseContainerd)
```

Comment:
We cannot move the additional parsing logic to `ProcessWithCallback` here since the entry needs to be parsed by `recombineParser` too. And the function params required for `recombineParser` are not accessible in a callback function.

Reply:
That's true. In theory, the call to the recombine's `Process` can stay where it is. Recombine's `Process` should be responsible for muting errors or not 🤔 ?